Python/pywarpx/_libwarpx.py

```diff
@@ -423,8 +423,18 @@ def initialize(self, argv=None, mpi_comm=None):
         doing anything else.
 
         '''
+        # no arguments passed? Take the ones from Python
         if argv is None:
             argv = sys.argv
+        # some arguments were passed
+        else:
+            # overwrite argv[0] with actual name
+            argv[0] = sys.argv[0]
```

Comment:
I would suggest that `argv[0]` should not be overwritten since it might be user input.

Instead, the place where this is called (in `WarpX.py`) can be changed so that it uses `sys.argv[0]` as the first element rather than `'warpx'`.  Also, perhaps the the `WarpX.init` routine can append `sys.argv` to the `argv` argument instead of doing it here.

Reply:
> I would suggest that `argv[0]` should not be overwritten since it might be user input.

`argv[0]` is always the process name. Currently we hard-code it to `warpx`.